pkg/agent/agentclient/clientset.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package agentclient
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"time"
+
+	"google.golang.org/grpc"
+	"k8s.io/klog"
+)
+
+// ClientSet consists of clients connected to each instance of an HA proxy server.
+type ClientSet struct {
+	mu      sync.Mutex              //protects the following
+	clients map[string]*AgentClient // map between serverID and the client
+	// connects to this server.
+
+	agentID     string // ID of this agent
+	address     string // proxy server address. Assuming HA proxy server
+	serverCount int    // number of proxy server instances, should be 1
```

Comment:
comment is incorrect, since value "0" is a valid case

Reply:
I don't think the comment is incorrect. "0" means it's not initialized yet.